validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -65,6 +67,8 @@
 public class ValidatorClientService extends Service {
 
   private static final Logger LOG = LogManager.getLogger();
+  private static final Duration DOPPELGANGER_DETECTION_CHECK_DELAY = Duration.ofSeconds(12);
+  private static final Duration DOPPELGANGER_DETECTION_TIMEOUT = Duration.ofMinutes(20);
```

Comment:
20 minutes seems a long time...
I'm not 100% sure how this is going to work, but I thought we'd be checking the keys we intend to load, and if they weren't seen as active then we'd be loading the keys, otherwise we'd be aborting before actually loading a key to be able to perform duties?
Presumably the check of keys should be fast-ish... like check last epoch, check current epoch, not seen == good to go...  This is a big assumption, im not sure of the design, so i could be wrong and we need 20 minutes, but i really hope it's not up to 20 minutes before we can load keys...